src/Catty/PlayerEngine/Sensors/Touch/FingerYSensor.swift

```diff
@@ -34,16 +35,16 @@
     }
     
     func rawValue() -> Double {
-        // TODO see FingerXSensor
-        return 0
+        guard let lastPosition = getTouchManager()?.lastPositionInScene() else { return type(of: self).defaultRawValue }
+        return Double(lastPosition.y)
     }
-
+    
     func convertToStandardized(rawValue: Double, for spriteObject: SpriteObject) -> Double {
-        // TODO see FingerXSensor
-        return rawValue
+        guard let _ = getTouchManager()?.lastPositionInScene() else { return type(of: self).defaultRawValue }
+        return -PositionYSensor.convertToStandardized(rawValue: rawValue, for: spriteObject)
```

Comment:
Please have a look if the PositionYSensor's conversion function is right.

Reply:
As already mentioned, please use a separate conversion function for this sensor.